BUILD.md

```diff
@@ -295,3 +298,59 @@ Module names can be found [here](bin/run_lint#L205) in the codebase.
   then you need to set `LD_LIBRARY_PATH` with:
   `source /opt/intel/sgxsdk/environment` .
   For details, see [PREREQUISITES](PREREQUISITES.md)
+
+# <a name="avaloncentos8"></a>Run Avalon on CentOS 8.2
+## <a name="sgxinstall"></a>SGX driver and PSW package installation
+1. Install SGX drivers
+   ```
+   cd /tmp
+   sudo yum update
+   sudo yum install kernel-devel gcc
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_linux_x64_driver_2.6.0_602374c.bin
+   chmod +x sgx_linux_x64_driver_2.6.0_602374c.bin
+   sudo ./sgx_linux_x64_driver_2.6.0_602374c.bin
+   
+   You should be able to see /dev/isgx device mapping on your host.
+   ```
+2. Install SGX PSW package
+   ```
+   cd /tmp
+   wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_rpm_local_repo.tgz
```

Comment:
Have we checked for a corresponding yum repository?

Reply:
I don't think it exists.